packages/server/modules/automate/graph/resolvers/automate.ts

```diff
@@ -451,11 +452,20 @@ export = {
       })
     },
     async trigger(parent, { automationId }, ctx) {
+      const automationRepository = createAutomationRepository({
+        db: {
+          AutomationTriggers: AutomationTriggers.knex,
```

Comment:
I think these `.knex` references are not working well enough now.

Reply:
I did what I needed to avoid changing all the db setup we already have in place
as long as this is scoped in a single place I think we can deal with it